File: Tipos de Dados, Variáveis e comandos de entrada e saída/exercicio5/exercicio5.js
/*Faça um programa que lê uma temperatura em
graus Celsius e a imprime convertida em graus
Fahrenheit. A fórmula de conversão é:

F ← (9*C+160)/5*/

const readline = require('readline')

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});
let celsius
rl.question('Digite a temperatura em Celsius: ', function(celsius){
    console.log(`A temperatura em Fahrenheit é: ${(9*celsius+160)/5}`)
    rl.close();
})

File: Tipos de Dados, Variáveis e comandos de entrada e saída/exercicio5/exercicio5.py
"""Faça um programa que lê uma temperatura em
graus Celsius e a imprime convertida em graus
Fahrenheit. A fórmula de conversão é:

F ← (9*C+160)/5"""

celsius = float(input('Digite a temperatura em celsius: '))
fahrenheit = ((9 * celsius + 160)/ 5)
print('A temperatura em Fahrenheit é: ',fahrenheit)